Samples/Tutorial Samples/WinMLPIXSample/WinMLPIXSample/main.cpp

```diff
@@ -0,0 +1,129 @@
+﻿#include "pch.h"
+#include "PixRedistMiddleware.h"
+#include "WinMLHelper.h"
+
+using namespace winrt;
+using namespace Microsoft::WRL;
+using namespace Microsoft::WRL::Wrappers;
+using namespace std;
+
+#ifndef USE_PIX
+#define USE_PIX
+#endif
+
+static hstring pixCaptureFilePath = L".\\PIXCapture\\capture.wpix";
+
+ID3D12Device* m_d3d12Device;
```

Comment:
"m_..." implies "member" variable, but that's not what these are, they're globals. And considering that you are manually releasing them at the end of the main method, maybe these should be in a class wrapped in a smart pointer that gets automatically cleaned up?

Reply:
Yeah, I will remove 'm_' prefix, I sort of purposefully avoided using smart pointer just so this works without having to add more dependencies.